deblur/workflow.py

```diff
@@ -550,6 +555,10 @@ def launch_workflow(seqs_fp, working_dir, read_error, mean_error, error_dist,
     with open(output_deblur_fp, 'w') as f:
         seqs = deblur(parse_fasta(output_msa_fp), read_error, mean_error,
                       error_dist, indel_prob, indel_max)
+        if seqs is None:
+            warnings.warn('multiple sequence alignment file %s contains no sequences' % output_msa_fp, UserWarning)
```

Comment:
You got me :) i disabled line length checking... think it makes the program less clear...
but fixed it anyway.